.aws/task-definition-template.json

```diff
@@ -23,6 +23,10 @@
         {
           "name": "REQUEST_QUEUE_URL",
           "valueFrom": "/tis/trainee/sync/${environment}/queue-url"
+        },
+        {
+          "name": "COGNITO_CONSULTATION_GROUP",
+          "valueFrom": "trainee-cognito-${environment}-v4-dsp-beta-consultation-group"
```

Comment:
Is this value the name or an id?
If it's the name we don't need to include the user pool version or environment.

Reply:
Related changes are updated on Ops: https://github.com/Health-Education-England/TIS-OPS/pull/528
The environment and version were removed. The only question from me is, now the Preprod and Prod have the same name of parameter, when it is applied for the second time, either error of parameter already existed would prompt or it would be overridden with the same value of the user group name. But this both sounds alright?